website/versions.json

```diff
@@ -1,6 +1,7 @@
 {
   "versions": {
-      "2.5.0": ["v2.5.0", "https://docs.openebs.io/"],
+      "2.6.0": ["v2.6.0", "https://docs.openebs.io/"],
```

Comment:
Couldn't find what is causing this version order issue.  Let's check once again after merging this PR.